test/worlds/hydrodynamics_flags.sdf

```diff
@@ -1,110 +1,108 @@
-        <p_gain>0.1</p_gain>
-      </plugin>
-
-      <plugin
-        filename="ignition-gazebo-joint-position-controller-system"
-        name="ignition::gazebo::systems::JointPositionController">
-        <joint_name>vertical_fins_joint</joint_name>
-        <p_gain>0.1</p_gain>
-      </plugin>
-
-      <plugin
-        filename="ignition-gazebo-thruster-system"
-        name="ignition::gazebo::systems::Thruster">
-        <namespace>tethys</namespace>
-        <joint_name>propeller_joint</joint_name>
-        <thrust_coefficient>0.004422</thrust_coefficient>
-        <fluid_density>1000</fluid_density>
-        <propeller_diameter>0.2</propeller_diameter>
-      </plugin>
-
-      <!-- Lift and drag -->
-
-      <!-- Vertical fin -->
-      <plugin
-        filename="ignition-gazebo-lift-drag-system"
-        name="ignition::gazebo::systems::LiftDrag">
-        <air_density>1000</air_density>
-        <cla>4.13</cla>
-        <cla_stall>-1.1</cla_stall>
-        <cda>0.2</cda>
-        <cda_stall>0.03</cda_stall>
-        <alpha_stall>0.17</alpha_stall>
-        <a0>0</a0>
-        <area>0.0244</area>
-        <upward>0 1 0</upward>
-        <forward>1 0 0</forward>
-        <link_name>vertical_fins</link_name>
-        <cp>0 0 0</cp>
-      </plugin>
-
-      <!-- Horizontal fin -->
-      <plugin
-        filename="ignition-gazebo-lift-drag-system"
-        name="ignition::gazebo::systems::LiftDrag">
-        <air_density>1000</air_density>
-        <cla>4.13</cla>
-        <cla_stall>-1.1</cla_stall>
-        <cda>0.2</cda>
-        <cda_stall>0.03</cda_stall>
-        <alpha_stall>0.17</alpha_stall>
-        <a0>0</a0>
-        <area>0.0244</area>
-        <upward>0 0 1</upward>
-        <forward>1 0 0</forward>
-        <link_name>horizontal_fins</link_name>
-        <cp>0 0 0</cp>
-      </plugin>
-
-      <plugin
-        filename="ignition-gazebo-hydrodynamics-system"
-        name="ignition::gazebo::systems::Hydrodynamics">
+      <link name="base_link">
+        <inertial>
+          <mass>147.8671</mass>
+          <inertia>
+            <ixx>3.000000</ixx>
+            <ixy>0</ixy>
+            <ixz>0</ixz>
+            <iyy>41.980233</iyy>
+            <iyz>0</iyz>
+            <izz>41.980233</izz>
+          </inertia>
+        </inertial>
+        <collision name="main_body_buoyancy">
+          <geometry>
+            <box>
+              <size>2 0.3 0.246445166667</size>
+            </box>
+          </geometry>
+        </collision>
+        <visual name='body_visual'>
+          <geometry>
+            <box>
+              <size>2 0.3 0.246445166667</size>
+            </box>
+          </geometry>
+        </visual>
+      </link>
+      <!-- Horizontal fins -->
+      <link name="horizontal_fins">
+        <pose>1.05 0 0 0 0 0</pose>
+        <inertial>
+          <pose>0 0 0 1.57 0 0.5</pose>
+          <mass>0.2</mass>
+          <inertia>
+            <ixx>0.0007924568755</ixx>
```

Comment:
We don't need so many moving parts. Just a box with a given mass of 147.8671kg should work. Same below.

Reply:
That's what I thought but the results seem to be different. [Here](https://gist.github.com/marcoag/4b661fc8bdbcead502d54a6dbc92c18d) is the world I used, I also tried adding the `0.2Kg*2` mass of the vertical and horizontal fins but still a different result. Any idea why?